src/domain/event/eventCategorisation/__tests__/EventCategorisation.test.tsx

```diff
@@ -38,25 +51,25 @@ test('renders ands display information correctly', () => {
 });
 
 test('show correct texts with different data', () => {
-  const newEventData = cloneDeep(eventData);
-  newEventData.event.keywords.push({
+  const newEventData = cloneDeep(event);
```

Comment:
I was thinking is cloneDeep needed. But maybe it is e.g. because of `newEventData.keywords.push`

Reply:
yeah just making sure that there is no probems if tests are added